File: QViz.NET/Periscope.QViz/JSON/APIResult.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test API Result Values
	/// </summary>
	public class APIResult
	{
		/// <summary>
		/// Identifier of the API Object
		/// </summary>
		public string apiId;
		
		/// <summary>
		/// Actual HTTP Status Code returned from the API Response
		/// </summary>
		public int actualHTTPStatus;
		
		/// <summary>
		/// Actual JSON String returned from the API Response
		/// </summary>
		public string actualJSONResult;
		
		/// <summary>
		/// Status of the Test Result (Pass, Fail, Broken)
		/// </summary>
		public string status;
		
		/// <summary>
		/// Error Message of the Test Result if it failed/broken
		/// </summary>
		public string error;
		
		/// <summary>
		/// Exception JSON String returned from the API Response
		/// </summary>
		public string errorJSONString;
		
		/// <summary>
		/// Version of the API
		/// </summary>
		public string apiVersion;
		
		/// <summary>
		/// Date and Time on which the Test execution was started
		/// </summary>
		public DateTime executionStartTime;
		
		/// <summary>
		/// Date and Time on which the Test execution was finished
		/// </summary>
		public DateTime executionEndTime;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public APIResult()
		{
			this.apiId = "";
			this.actualHTTPStatus = 0;
			this.actualJSONResult = "";
			this.status = "";
			this.error = "";
			this.errorJSONString = "";
			this.apiVersion = "";
			this.executionStartTime = DateTime.Now;
			this.executionEndTime = DateTime.Now;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/APIResult.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for Test API Result Values
 */
public class APIResult {

	/**
	 * Identifier of the API Object
	 */
	public String apiId;

	/**
	 * Actual HTTP Status Code returned from the API Response
	 */
	public Integer actualHTTPStatus;

	/**
	 * Actual JSON String returned from the API Response
	 */
	public String actualJSONResult;

	/**
	 * Status of the Test Result (Pass, Fail, Broken)
	 */
	public String status;

	/**
	 * Error Message of the Test Result if it failed/broken
	 */
	public String error;

	/**
	 * Exception JSON String returned from the API Response
	 */
	public String errorJSONString;

	/**
	 * Version of the API
	 */
	public String apiVersion;

	/**
	 * Date and Time on which the Test execution was started
	 */
	public Date executionStartTime;

	/**
	 * Date and Time on which the Test execution was finished
	 */
	public Date executionEndTime;

	/**
	 * Default Constructor
	 */
	public APIResult() {
		this.apiId = "";
		this.actualHTTPStatus = 0;
		this.actualJSONResult = "";
		this.status = "";
		this.error = "";
		this.errorJSONString = "";
		this.apiVersion = "";
		this.executionStartTime = null;
		this.executionEndTime = null;
	}

}
